src/test/resources/testData.json

```diff
@@ -7,6 +7,11 @@
       "password": "eotTesting@0",
       "firstName": "Eot",
       "lastName": "Testing"
+    },
+    "aman.daharwal": {
```

Comment:
This is incorrect. 
The data is per environment. If you need to create user-specific data, then 
1. you need to add it for every environment
2. you need to update existing data to also fit in that pattern